web/README.md

```diff
@@ -0,0 +1,59 @@
+# User Overrides Web Panel
+Communicate with the vehicle's CAN bus using a web application. Send custom or preset
+messages through the interactive and user-friendly panel.
+
+## Requirements
+The web panel is ran through the Flask web framework, which comes bundled to
+the default Python installation.
+
+In addition, the application depends on several web sockets to communicate
+with the server. Install them using:
+
+```bash
+pip install flask-socketio
+pip install gevent-websocket
+```
+
+Version inter-compatibility is highly limited, which may lead to several difficulties
+while running the web application. Visit the [version compatibility](https://flask-socketio.readthedocs.io/en/latest/#version-compatibility)
+if encountering such errors.
+
+The application uses [can_data/CAN2.dbc](https://github.com/msfrt/PiLogger/blob/web/web/can_data/CAN2.dbc)
+as database to extract specifications of signals and messages.
+
+## Server Set-Up
+In order to set-up the web server, run the following command in the folder
+where the app.py file is located at the RasPi.
+
+```bash
+flask run --host=0.0.0.0
+```
+
+## Usage
+Once set-up, access the web application [here](http://147.92.111.86:5000/).
+
+### Sending a Message Manually
+Input the ID of any message you want to send to the bus, and include the encoded
+(max.) 8-bytes-long data in hexadecimal. Then click send.
+
+### USER_11 Message
+I.e., setting the display's LED color. Input 0 for default color, 1 for
+blinking white, and 2 for blinking yellow. Then click send.
+
+### USER_12 Message
```

Comment:
It would be cool to include an annoted picture that explains what is what. You can use one of the photos that I've sent you previously. Simply google "how too include an image in markdown" and there should actually be a GitHub guide on how to do so. You can just create a new directory called `images` to store the image in. SR-Libraries has a few examples of using images if you scrounge around. I know for sure the aero map script does.